analysis/study_definition.py

```diff
@@ -1,41 +1,55 @@
 
-# Import functions
-
 from cohortextractor import (
-    StudyDefinition, 
-    patients, 
-    codelist, 
-    codelist_from_csv
+    StudyDefinition,
+    patients,
+    codelist_from_csv,
+    codelist,
+    filter_codes_by_category,
+    combine_codelists,
 )
 
-# Import codelists
+# Important Dates
+campaign_start = "2020-12-07" # change this if you need to
+latest_date = "2021-01-13" # for expectations only atm
```

Comment:
this should be auto updating if possible?

Reply:
@sebbacon we discussed this briefly in tech call. Not sure if you have any ideas about this. We talked about using "today" but that not necessarily being ideal either. 

@evansd as well

Reply:
this issue is related https://github.com/opensafely/cohort-extractor/issues/420. If we can get the latest data import dates then we can use, say, `latest_date = S1_latest_import_date - 4 days`